python/ray/tune/schedulers/pbt.py

```diff
@@ -161,7 +168,8 @@ def __init__(self,
                  perturbation_interval=60.0,
                  hyperparam_mutations={},
                  resample_probability=0.25,
-                 custom_explore_fn=None):
+                 custom_explore_fn=None,
+                 log_config=False):
```

Comment:
We can always do this by default? That would avoid the need for a flag.

Reply:
Do you mean we should always do it and remove the flag, or have it set to a default_true setting?